owncloudApp/src/main/res/values/strings.xml

```diff
@@ -232,7 +232,7 @@
     <string name="pass_code_configure_your_pass_code_explanation">The passcode will be requested every time the app is started</string>
     <string name="pass_code_reenter_your_pass_code">Please reenter your passcode</string>
     <string name="pass_code_remove_your_pass_code">Remove your passcode</string>
-    <string name="pass_code_mismatch">The passcodes are not the same</string>
+    <string name="pass_code_mismatch">The passcodes are not the same\n Enter your password</string>
```

Comment:
I would keep the error and title in different strings. I would not mix them. Showing the error in a different field would do the job. What do you think?

Reply:
Ok, I will take the approach you suggested in the comment so that these two requested changes are applied too 🍻